stable/rabbitmq/values-production.yaml

```diff
@@ -146,6 +146,15 @@ rabbitmq:
   ## Use this as additional configuraton in classic config format (Erlang term configuration format)
   advancedConfiguration: |-
 
+  ## Enable encryption to rabbitmq
+  ssl:
+    enabled: false
```

Comment:
Why are you disabling it in the production values but enabling it in the regular values?

Reply:
is a mistake

Reply:
is a mistake